osbuild/main_cli.py

```diff
@@ -136,8 +136,9 @@ def osbuild_cli() -> int:
     exports = set(args.export)
     unresolved = [e for e in exports if e not in manifest]
     if unresolved:
+        available = ",".join(manifest.pipelines.keys())
         for name in unresolved:
-            print(f"Export {vt.bold}{name}{vt.reset} not found!")
+            print(f"Export {vt.bold}{name}{vt.reset} not found in {available}")
```

Comment:
Do f-strings use `str` or `repr` by default? Maybe making it explicit with `{available!s}` is an idea?

Reply:
I got curious :) It uses `__format__` which is essentially `str()` and they even have a section about them being redundan - https://peps.python.org/pep-0498/#s-r-and-a-are-redundant